docs/ops/arithmetic/Divide_1.md

```diff
@@ -4,48 +4,51 @@
 
 **Category**: Arithmetic binary operation
 
-**Short description**: *Divide* performs element-wise division operation with two given tensors applying multi-directional broadcast rules.
+**Short description**: *Divide* performs element-wise division operation with two given tensors applying broadcasting rule specified in the *auto_broacast* attribute.
+
+**Detailed description**
+Before performing arithmetic operation, input tensors *a* and *b* are broadcasted if their shapes are different and `auto_broadcast` attribute is not `none`. Broadcasting is performed according to `auto_broadcast` value.
+
+After broadcasting *Divide* performs divison operation for the input tensors *a* and *b* using the formula below:
+
+\f[
+o_{i} = a_{i} / b_{i}
+\f]
```

Comment:
add a comment about division by zero. Refer to FloorMod and Mod for the example.

Reply:
done